x/axelarcork/keeper/keeper.go

```diff
@@ -248,13 +248,13 @@ func (k Keeper) GetWinningAxelarCork(ctx sdk.Context, chainID uint64, contractAd
 }
 
 func (k Keeper) DeleteWinningAxelarCorkByBlockheight(ctx sdk.Context, chainID uint64, blockHeight uint64, cork types.AxelarCork) {
-	ctx.KVStore(k.storeKey).Delete(types.GetWinningAxelarCorkKey(chainID, blockHeight, common.HexToAddress(cork.TargetContractAddress), cork.Deadline))
+	ctx.KVStore(k.storeKey).Delete(types.GetWinningAxelarCorkKey(chainID, blockHeight, common.HexToAddress(cork.TargetContractAddress)))
 }
 
 // TODO (Collin): Need pruning logic. This method is unused.
 func (k Keeper) DeleteWinningAxelarCork(ctx sdk.Context, chainID uint64, c types.AxelarCork) {
```

Comment:
Do you want to handle pruning winning corks in this PR?

Reply:
I think it's unused because they get deleted here:

https://github.com/PeggyJV/sommelier/blob/55929891728ac0c21625bd23970b9c59c1f10843/x/axelarcork/keeper/packet.go#L72